suite-native/transactions/src/components/TransactionsList/TransactionListPagination.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { Box, Button } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+type TransactionListPaginationProps = {
+    totalAccountsTransactions?: number;
+    page: number;
+    onSetPage: (page: number) => void;
+};
+
+export const TRANSACTIONS_PER_PAGE = 5;
+
+const paginationWrapperStyle = prepareNativeStyle(_ => ({
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'center',
+}));
+
+export const TransactionListPagination = ({
+    totalAccountsTransactions,
+    page,
+    onSetPage,
+}: TransactionListPaginationProps) => {
+    const { applyStyle } = useNativeStyles();
+    if (!totalAccountsTransactions) return null;
+
+    const numberOfPages = Math.ceil(totalAccountsTransactions / TRANSACTIONS_PER_PAGE);
+
+    const paginationElements = [...Array(numberOfPages)].map((_, index) => index + 1);
+
+    return (
+        <Box style={applyStyle(paginationWrapperStyle)}>
+            {paginationElements.map(el => (
```

Comment:
I would really appreciate it if we didn't use shortcuts 🙏 

```suggestion
            {paginationElements.map(element => (
```

Reply:
This file got deleted so it's not an issue anymore. Product issue changed a little bit as well, so I am changing this back to draft to do some more changes.